### ISSUE TITLE
How to fix CA1416 for net5-windows10.0.17763.0 ?

### ISSUE THREAD

**Issue by bgavrilMS** [dotnet/roslyn-analyzers#4932]
### Analyzer

**Diagnostic ID**:  CA1416 

### Analyzer source

**SDK**: [Built-in CA analyzers in .NET 5 SDK or later](https://docs.microsoft.com/dotnet/fundamentals/productivity/code-analysis)

**Version**: 5.0.200-preview.21077.7 

### Describe the bug

In my library I have
```csharp
    public class MSAL
    {
        [SupportedOSPlatform("windows10.0.17763.0")]
        public static void UseWAM()
        {
            WebTokenRequest webTokenRequest = new WebTokenRequest(null);
        }
    }
```
A calling app does :

```csharp
    class Program
    {
        static void Main(string[] args)
        {
            if (OperatingSystem.IsWindowsVersionAtLeast(10, 0, 17763, 0))
            {
                MSAL.UseWAM(); // gets a warning despite the platform guard
            }
       }
   }
```

### Expected behavior
no CA1416 warning in app

### Actual behavior
warning

## Additional context

If I change it to [SupportedOSPlatform("windows")] and OperatingSystem.IsWindows, the warning goes away.

**Comment by buyaa-n** [comment]
hm, `OperatingSystem.IsWindowsVersionAtLeast(10, 0, 17763, 0)` is evaluated to version `10.0.17763` the comparison operators of the Version type is a bit strange
```
Console.WriteLine(Version.Parse("10.0.17763.0") < Version.Parse("10.0.17763")); // False
Console.WriteLine(Version.Parse("10.0.17763.0") > Version.Parse("10.0.17763")); // True
Console.WriteLine(Version.Parse("10.0.17763.0") == Version.Parse("10.0.17763")); // False
```
as a workaround @bgavrilMS could you remove leading `0` from the attribute ([SupportedOSPlatform("windows10.0.17763")]) until it gets resolved?

**Comment by buyaa-n** [comment]
Fixed with https://github.com/dotnet/roslyn-analyzers/pull/4943
